### DISCUSSION TITLE
Minor note

### DISCUSSION THREAD

**Discussion by ozrien** [HuskieRobotics/3061-lib#55] [General]
This looks great...

https://github.com/HuskieRobotics/3061-lib/blob/d6ba072639068f1684528705d3cf698821f262dd/src/main/java/frc/lib/team3061/swerve/SwerveModuleIOTalonFXPhoenix6.java#L265
... its worth mentioning that even if you do set `EnableFOC` to true and device is not licensed, motor will still spin okay (without FOC).
This is true for all DutyCycle and Voltage output types (but not TorqueCurrent since there is no nonFOC version of this).

There's also a [stickyfault](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/hardware/core/CoreTalonFX.html#getStickyFault_UnlicensedFeatureInUse()) that tells if you this happens.

Just though that was useful for future if you are troubleshooting an issue.

**Comment by gcschmit** [comment]
> ... its worth mentioning that even if you do set `EnableFOC` to true and device is not licensed, motor will still spin okay (without FOC).
> This is true for all DutyCycle and Voltage output types (but not TorqueCurrent since there is no nonFOC version of this).
> There's also a [stickyfault](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/hardware/core/CoreTalonFX.html#getStickyFault_UnlicensedFeatureInUse()) that tells if you this happens.
> 
> Just though that was useful for future if you are troubleshooting an issue.

Thank you! The FaultReporter class does check a variety of faults, including using unlicensed features, and logs alerts to inform the user.

Now we can focus all of efforts on incorporating all the new features in the Phoenix 6 beta!
